test/TEST-58-REPART/test.sh

```diff
@@ -14,6 +14,7 @@ test_append_files() {
         install_dmevent
         instmods dm_verity =md
         instmods erofs
+        inst_binary setpriv
```

Comment:
Please unconditionally install setpriv, i.e. move the line above `if ! get_bool ...` line, otherwise the test fails on Ubuntu CIs.